builds/jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -70,15 +73,14 @@ def getEnvironmentLogicalId() {
 	return g_environment_logical_id
 }
 
-def getEnvironmentInfo() {
+def getEnvironmentInfo(g_environment_logical_id) {
```

Comment:
`g_environment_logical_id` is a global var which is assumed to be available already (see `getEnvDeploymentDescriptor`). Do we need to pass this var?

Reply:
"g_environment_logical_id"  is not initialized in the case of master branch deployment. Also the "physical_id" is same for both stg and prod environments, we need to pass the logical id to get the exact environment details.  "g_environment_logical_id" I will change it to a local variable.

Reply:
Changed to local variable

Reply:
Had found another work around for this.  Please see the latest change.